src/HighTable.tsx

```diff
@@ -280,29 +305,13 @@ export default function HighTable({
 
         // Await all pending promises
         await Promise.all(rowsChunk.flatMap(asyncRow => [asyncRow.index, ...Object.values(asyncRow.cells)]))
-
-        // if user scrolled while fetching, fetch again
```

Comment:
"if user scrolled while fetching, fetch again"

I'm a little nervous deleting this and the `pendingUpdate` mechanism? Maybe it's not needed because we always call `fetchRows`? I guess this was from a time when we locked only one fetch at a time, but now we allow multiple fetches at once, but throttled? I guess its okay to delete this then.

Reply:
re pendingUpdate: it was not used anymore: its value is always false (not sure when it was disabled, maybe when pendingRequest was introduced?)